src/main/java/by/georgprog/epicmusicstore/service/album/AlbumServiceImpl.java

```diff
@@ -0,0 +1,101 @@
+package by.georgprog.epicmusicstore.service.album;
+
+import by.georgprog.epicmusicstore.config.security.UserDetailsImpl;
+import by.georgprog.epicmusicstore.dto.AlbumDto;
+import by.georgprog.epicmusicstore.exeption.forbidden.ObtainingDataException;
+import by.georgprog.epicmusicstore.exeption.unauthorized.AlbumNotFoundException;
+import by.georgprog.epicmusicstore.exeption.unauthorized.UserNotFoundException;
+import by.georgprog.epicmusicstore.mapper.AlbumMapper;
+import by.georgprog.epicmusicstore.model.AlbumEntity;
+import by.georgprog.epicmusicstore.repo.AlbumRepository;
+import by.georgprog.epicmusicstore.service.user.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class AlbumServiceImpl implements AlbumService {
+
+    private final UserService userService;
```

Comment:
Let's use UserRepository here. I want services to not use each other because it may be a reason of cycle dependencies. Let's do the following:
Controllers --use--> Services --use--> Repositories and Mappers (and some other services which are not connected with entities and DB)